elide-core/src/main/java/com/yahoo/elide/ElideSettingsBuilder.java

```diff
@@ -45,8 +45,8 @@
     private List<JoinFilterDialect> joinFilterDialects;
     private List<SubqueryFilterDialect> subqueryFilterDialects;
     private Map<Class, Serde> serdes;
-    private int defaultMaxPageSize = Pagination.MAX_PAGE_LIMIT;
-    private int defaultPageSize = Pagination.DEFAULT_PAGE_LIMIT;
+    private int defaultMaxPageSize = PaginationImpl.MAX_PAGE_LIMIT;
+    private int defaultPageSize = PaginationImpl.DEFAULT_PAGE_LIMIT;
```

Comment:
Should these constants be defined in the interface?

Reply:
👍